### ISSUE TITLE
(Automated) Testing of existing notebooks

### ISSUE THREAD

**Issue by abarciauskas-bgse** [developmentseed/cloud-optimized-geospatial-formats-guide#44]
Most of the example notebooks have been created by myself, @wildintellect and @kylebarron. We need someone to go through and make sure the instructions to run the notebooks make sense and are able to run without error.

Bonus: Automate testing the notebooks through a github runner. We have done something like this for MAAP in the past.

**Comment by kylebarron** [comment]
When I was creating the environment files in https://github.com/developmentseed/cloud-optimized-geospatial-formats-guide/pull/43 I was able to run through all the notebooks without error, except for the end of the GeoParquet notebook due to versioning issues.

I made https://github.com/conda-forge/gdal-feedstock/pull/805 which may fix the `libgdal-arrow-parquet` versioning issues

**Comment by abarciauskas-bgse** [comment]
Thanks @kylebarron - do you think we should still keep this ticket open for someone else to go through the instructions and / or automate the tests?

**Comment by kylebarron** [comment]
Yeah! We should keep it open for the automated testing. I've never worked with notebooks on CI, so not sure if you have standard practices for that. This did also remind me of the https://github.com/nteract/papermill project, which could be useful if we ever want to parametrize the notebooks we're running
